index.js

```diff
@@ -44,12 +44,15 @@ type Section {
   pages: [Page]
   questionnaire: Questionnaire
   position: Int!
-  introduction: Introduction
+  introductionTitle: String
+  introductionContent: String
+  introductionEnabled: Boolean!
+  introduction: Introduction @deprecated(reason: "Use Section fields instead")
 }
 
 type Introduction {
```

Comment:
Think we're also going to need to deprecated this type so we can clean it up later.

Reply:
Removed on deprecated fields branch